tests/conftest.py

```diff
@@ -21,3 +23,18 @@ def db():
     yield params
 
     cursor.execute(f"DROP DATABASE {test_db_name}")
+
+
+@pytest.fixture()
+def fake_db(mocker):
+    @contextmanager
+    def execute(response):
+        yield response
+
+    def _fake_db(response=None):
+        execute_mock = mocker.patch(
+            "septentrion.db.execute", return_value=execute(response)
+        )
+        return execute_mock
+
+    return _fake_db
```

Comment:
Didn't notice it in the other PR, but I believe this fixture could be simplified:
```
# definition: 
@pytest.fixture()
def fake_db(mocker):
	yield mocker.patch(
        "septentrion.db.execute", return_value=execute(response)
    )
# usage when you don't need a return value:
def test_something(fake_db):
    ...
# usage when you need a return value:
def tests_somethong(fake_db):
    fake_db.return_value = [True]
    ...
```

I may completely miss the point, though...?